### PR TITLE
Show different text for answer

### PR DESCRIPTION
**Pull request by Sweetpurple616**
On toggleSelected add answer to preValue and on handleSubmit add answer to answers in value.
In Results added the context, loops through the answers and uses the answer to show the right text in content

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -30,11 +30,11 @@ const App = () => {
       setStyle({ width: (width <= 100 ? width : 100) + "%" })
   }
 
-  const handleStyle = () => {
-    const width = (100 / (length + 1)) * (index + 1);
-    // Not sure if the if statement in line 25 works
-    setStyle({ width: width <= 100 ? width : 100 + "%" });
-  };
+  // const handleStyle = () => {
```
Comment (janesssa) [inline src/App.js:33]: Why is this outcommented?

File: src/components/Header.js [hunk 2]
```diff
@@ -9,8 +9,6 @@ const Header = ({ text, style, handleNext}) => {
     <>
       <div className="header">
         <h3>product naam</h3>
-        <small>{JSON.stringify(preValue)}</small>
```
Comment (janesssa) [inline src/components/Header.js:12]: Sorry about this, thanks!

File: src/components/Question.js [hunk 3]
```diff
@@ -15,28 +15,29 @@ const Question = ({content, index, handleNext}) => {
     const {value, setValue} = useContext(ValueContext)
     const {preValue, setPreValue} = useContext(PreValueContext)
 
-    console.log(question.answers)
+    // console.log(question.answers)
     const refs = useRef(question.answers.map(() => createRef()))
-    console.log(refs)
+    // console.log(refs)
```
Comment (janesssa) [inline src/components/Question.js:20]:
```suggestion
```

File: src/components/Question.js [hunk 4]
```diff
@@ -15,28 +15,29 @@ const Question = ({content, index, handleNext}) => {
     const {value, setValue} = useContext(ValueContext)
     const {preValue, setPreValue} = useContext(PreValueContext)
 
-    console.log(question.answers)
+    // console.log(question.answers)
```
Comment (janesssa) [inline src/components/Question.js:18]:
```suggestion
```

File: src/components/Question.js [hunk 5]
```diff
@@ -15,28 +15,29 @@ const Question = ({content, index, handleNext}) => {
...
     const refs = useRef(question.answers.map(() => createRef()))
-    console.log(refs)
+    // console.log(refs)
 
     useEffect(() => {
         
     }, [length])
 
     const toggleSelected = (i, v, c, cn) => {
+        console.log(i)
```
Comment (janesssa) [inline src/components/Question.js:27]:
```suggestion
```

File: src/components/Question.js [hunk 6]
```diff
@@ -15,28 +15,29 @@ const Question = ({content, index, handleNext}) => {
...
         if(refs.current[i]){
             const activeRef = refs.current[i].current;
-            console.log(activeRef)
+            // console.log(activeRef)
```
Comment (janesssa) [inline src/components/Question.js:30]:
```suggestion
```

File: src/components/Question.js [hunk 7]
```diff
@@ -15,28 +15,29 @@ const Question = ({content, index, handleNext}) => {
...
             if(activeRef !== null) {
                 activeRef.classList.add(cn)
                 refs.current.map(ref => {
-                    console.log(ref)
+                    // console.log(ref)
```
Comment (janesssa) [inline src/components/Question.js:34]:
```suggestion
```

File: src/components/Question.js [hunk 8]
```diff
@@ -45,21 +46,24 @@ const Question = ({content, index, handleNext}) => {
         let pa = preValue.value
         let category = preValue.category
         setValue((v) => {
+            console.log(v)
```
Comment (janesssa) [inline src/components/Question.js:49]:
```suggestion
```

File: src/components/Question.js [hunk 9]
```diff
@@ -45,21 +46,24 @@ const Question = ({content, index, handleNext}) => {
...
+                    [category]: categoryValue,
+                // answers : answers
                 }
             }
         })
         refs.current.map(ref => {
             const classList = ref.current.classList
             if(classList.contains('answer-img')) {
-                console.log('remove img clas')
+                // console.log('remove img clas')
```
Comment (janesssa) [inline src/components/Question.js:66]:
```suggestion
```

File: src/App.js (hunk 1 above)
Comment (Sweetpurple616) [reply src/App.js:33]: Got a error that is was double

**Pull request merged**